src/EcologicalNetworks.jl

```diff
@@ -74,6 +74,9 @@ include(joinpath(".", "rand/null.jl"))
 include(joinpath(".", "rand/linfilter.jl"))
 export linearfilter, linearfilterzoo
 export null1, null2, null3, null4
+include(joinpath(".", "rand/RDPG.jl"))
```

Comment:
Assuming they are mostly used together, you can drop them in a single file.

Reply:
I will take the SVD part out, as we might want to add the code that @TanyaS08 wrote for her paper as well